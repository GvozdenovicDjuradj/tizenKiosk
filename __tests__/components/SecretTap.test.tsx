import React from "react"
import SecretTap from "../../src/components/secretTap"
import renderer from "react-test-renderer"
import { createState } from "../fixtures"
import { Provider} from "react-redux"
import mockStore from "redux-mock-store"

describe("Secret tap tests", () => {
  it("should render Secret tap area successfully without template", () => {
    const state = createState()
    const store = mockStore()(state)
    const wrapper = renderer.create(
      <Provider store={store}>
        <SecretTap />
      </Provider>
    )
    expect(wrapper.toJSON()).toMatchSnapshot()
  })
})
