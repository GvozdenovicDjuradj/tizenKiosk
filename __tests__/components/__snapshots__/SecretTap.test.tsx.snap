// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Secret tap tests should render Secret tap area successfully without template 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "height": "100%",
      "left": 0,
      "opacity": 1,
      "position": "absolute",
      "top": 0,
      "width": 80,
    }
  }
/>
`;
